OpenRA.Mods.Common/Widgets/Logic/Ingame/MusicControllerLogic.cs

```diff
@@ -9,37 +9,59 @@
  */
 #endregion
 
+using System.Collections.Generic;
 using System.Linq;
 using OpenRA.GameRules;
 using OpenRA.Graphics;
+using OpenRA.Mods.Common.Lint;
 using OpenRA.Mods.Common.Traits;
 using OpenRA.Widgets;
 
 namespace OpenRA.Mods.Common.Widgets.Logic.Ingame
 {
+	[ChromeLogicArgsHotkeys("StopKey", "PauseKey", "PrevKey", "NextKey")]
```

Comment:
for consistency, it would be better here and in .yaml to keep definition order like so far in other places